Makefile

```diff
@@ -20,15 +20,21 @@ clean_deb_source :
 	rm -rf net-snmp-${DEBIANVER}/
 	-rm .*.built
 
-${DEBS} : .clearwater1.built
+net-snmp-5.7.2~dfsg/COPYING:
+	apt-get source net-snmp=${DEBIANVER}-${UBUNTUVER}
+
+${DEBS}: .all_built
 
 .%.built : %.patch
 	touch $@.tmp
-	apt-get source net-snmp=${DEBIANVER}-${UBUNTUVER}
 	patch -p1 < $<
+	mv $@.tmp $@
+
+.all_built: net-snmp-5.7.2~dfsg/COPYING .clearwater1.built .clearwater2.built
```

Comment:
Also do you not have to reset the code-base before building the second patch?  Maybe you need some targets for `.clearwater%.applied` and then each build is followed immediately by a `debian/rules clean`?